packages/wallet/src/services/HandlesTracker.ts

```diff
@@ -83,5 +128,19 @@ export const createHandlesTracker = ({ tip$, assetInfo$, handlePolicyIds, logger
       );
     }),
     distinctUntilChanged((a, b) => sameArrayItems(a, b, handleInfoEquals)),
+    map((handles) =>
+      from(handles).pipe(
+        concatMap((handle) =>
+          of(hydrateHandle(handleProvider, logger)(handle)).pipe(
+            delay(RESOLVE_HANDLE_DELAY),
```

Comment:
What is the purpose of `delay` here?

Reply:
The ada handle API has a rate limit and I was getting 429 errors when fetching without the delay.

Reply:
This is a bit fragile. Ideally we would associate some type of retry with backoff only when encountering this error. Are there alternatives @mkazlauskas ?

Reply:
Fixed here: https://github.com/input-output-hk/cardano-js-sdk/pull/786/commits/2ecb53301b9c37c6727088f9fe3ac58c4d2592db.

Reply:
Is 10ms enough for initial interval? Originally the delay was 50

Reply:
I forgot to link this last update from the last round: https://github.com/input-output-hk/cardano-js-sdk/pull/786/commits/8eee4376e6d032c8e55c5ab03942bc15d1afcf56